updater/xml_to_sql/post_processing.py

```diff
@@ -246,11 +246,76 @@ def trim_rawassignee(config):
         (name_last IS NULL) AND
         (organization IS NULL);""")
 
+def fix_rawassignee_wrong_org(config):
```

Comment:
Is there a risk of false positives here? i.e. records we thought were organization was in reality just someone with just first name. ( https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/ ) . If the existing data supports the theory that in _patent_ data, this seems to be the case most/all of the time, then I would suggest creating "audit" records to keep track of which records were changed in post processing.